wip (#48)
apostrophecms/import-export: CHANGELOG.md

## UNRELEASED

### Adds

> * Import nows detects if the locale found in the exported docs is different from the current site one.

Reviewer: "Import now..."

Author: @BoDonkey done

> * Import nows detects if the locale found in the exported docs is different from the current site one.  
> If the site has only one locale configured, then the docs are automatically re-written with the site locale.  
> If the site has multiple locales configured, the user is given the possibility to abort the import or re-write the docs with the site locale.

Reviewer: ```suggestion
* Import now detects if the locale found in the exported docs is different from the current site one. If the site has only one locale configured, then the docs are automatically re-written with the site locale. If the site has multiple locales configured, the user can abort the import or re-write the docs with the site locale.
```
